src/backend/web.rs

```diff
@@ -142,6 +142,7 @@ impl crate::RenderInner<Context> for RenderPass {
     fn set_index_buffer(
         &mut self,
         buffer: &Sendable<web_sys::GpuBuffer>,
+        index_format: wgt::IndexFormat,
```

Comment:
this will trigger a warning about unused variable

Reply:
Yes, I purposely left it like that as a reminder of the web-sys update.